test/scripts/hello_world_attest_fn_call.txtar

```diff
@@ -0,0 +1,10 @@
+# [execute] call the bky-as CLI to attest a function call
+stdin fn-call.json
+exec bky-as attest-fn-call
+cp stdout result.json
+
+# [check] assert stderr and stdout expected values
+exec jq -r '.transitive_attested_function_call.claims.output | @base64d' result.json
+stdout 'Hello, World!'
+exec jq -r '.transitive_attested_function_call.logs | @base64d' result.json
+stdout 'Writing "Hello, World!" to host'
```

Comment:
here we see a little more of the testscript DSL. `stdout 'pattern'` just checks that the pattern is found in stdout, allowing us to assert some expected string/output

Reply:
this testscript is slightly different from coingecko because the coingecko example uses a result type which needs to be parsed first. This example writes a string directly to the function output.